I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 251, in script
    jump escnea_2_cap1
ScriptError: could not find label 'escnea_2_cap1'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 251, in script
    jump escnea_2_cap1
  File "C:\Users\Camilo\Downloads\renpy-8.3.2-sdk\renpy\ast.py", line 1699, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\Camilo\Downloads\renpy-8.3.2-sdk\renpy\script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'escnea_2_cap1'.

Windows-10-10.0.19041 AMD64
Ren'Py 8.3.2.24090902
AALeph 0.06
Thu Oct 24 13:55:12 2024
